python/setup2.py

```diff
@@ -11,7 +11,7 @@ def read_requirements(path):
 setup(
```

Comment:
you weren't supposed to have this file.

Reply:
File has been removed with subsequent commit.